### ISSUE TITLE
About RagChecker  

### ISSUE THREAD

**Issue by binghangli378** [explodinggradients/ragas#1069]
Recently, I have noticed something similar to ragas, which is the [RagChecker](https://github.com/amazon-science/RAGChecker). It provides a new perspective to evaluate RAG pipelines which separately focuses on:
- Relevant chunk
- Irrelevant chunk used by the model
- Irrelevant chunk ignored by the model

This new perspective will provide a more detailed evaluation of the model’s performance, allowing for a deeper understanding of how different types of data chunks impact the evaluation process.

I am willing to design some new evaluation methods based on it. Please let me know if you are open to this idea, and I can provide further assistance or code examples.

**Comment by jjmachan** [comment]
@binghangli378 that is a very interesting Idea, reopening this to track more. Would you still like to help out on this?

@shahules786 something we can consider for #1010 ?

**Comment by shahules786** [comment]
@binghangli378 Yes, this is very interesting. I will check this out today and share my suggestions. @jjmachan

**Comment by sky-2002** [comment]
I can take up `noise-sensitivity`

**Comment by shahules786** [comment]
@sky-2002 Sure, Can you please raise an issue for this so that we can keep track of it? I'll assign it to you if you DM me.

**Comment by anistark** [comment]
Closing as completed.
